motoman_gp215_250_support/package.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>motoman_gp215_250_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>ROS-Industrial support for the Motoman GP215 and GP250 (variants).</p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for Motoman GP215/250 manipulators.
+    </p>
+    <p>
+      <b>Specifications</b>
+    </p>
+    <ul>
+      <li>GP215 - Default (YR-1-06VX215-A00, YR-1-06VX215-A20)</li>
+	  <li>GP250 - Default (YR-1-06VX250-A00, YR-1-06VX250-A20)</li>
+   </ul>
+    <p>
+      Joint limits and maximum joint velocities are based on the information 
+      found in:
+      <em>YASKAWA MOTOMAN-GP215,GP250 INSTRUCTIONS (HW2480629)</em>
+      version <em>194804-1CD, rev 1</em>.
+	  <em>YASKAWA MOTOMAN-GP215,GP250,GP280 INSTRUCTIONS (HW1484400)</em>
+      version <em>182639-1CD, rev 2</em>.
```

Comment:
@EricMarcil: could you clarify which of these documents you used?

Both, for both robots, or the first for the 215 and the second for the 250? Something else?

Reply:
I used both for both robots. The information affecting URDF model is identical in both.
The difference is that 182639-1CD is for the standard model -A00 and the 194804-1CD is for the -A20 which is the Dust-Proof/Water-Proof version with higher IP rating.